app/assets/stylesheets/application/controllers/works.scss

```diff
@@ -148,4 +148,18 @@
       margin-top: 20px;
     }
   }
+
+  .info {
+    .new-edit-request-btn {
+      margin: 14px 0 10px 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `14px 0 10px` instead of `14px 0 10px 0`